my_view_inc.php

```diff
@@ -342,7 +342,9 @@
 	<td class="nowrap width-13 my-buglist-id">
 		<?php
 			print_bug_link( $t_bug->id, false );
-
+			if( $t_bug->votes > 1 ) {
+				echo ' ^', $t_bug->votes;
```

Comment:
Currently, no UI use term "vote". Only internal code operates with "votes". Therefore, it just looks a bit weird to me to have something like +100. I would rather delay this until real voting will be implemented, but I'll be fine either way.